pulser/devices/_device_datacls.py

```diff
@@ -48,6 +48,7 @@ class Device:
     min_atom_distance: int
     _channels: tuple[tuple[str, Channel], ...]
     interaction_coeff: float = 5008713.
+    C_3 = 3700.
```

Comment:
For clarity these should be called `interaction_coeff_ising` and `interaction_coeff_xy`